images/win/Windows2019-Azure.json

```diff
@@ -446,8 +446,8 @@
         {
             "type": "powershell",
             "environment_vars": [
-                "BOOST_VERSIONS=1.69.0",
-                "BOOST_DEFAULT=1.69.0"
+                "BOOST_VERSIONS=1.69.0,1.72.0",
```

Comment:
Boost 1.69 doesn't support VS19
https://github.com/actions/virtual-environments/issues/74

Reply:
But it does support VS19, according to the Canary tests.
https://buildcanary.visualstudio.com/PipelineCanary/_build/results?buildId=755795